nncf/quantization/quantize.py

```diff
@@ -26,7 +26,7 @@ def quantize(model: ModelType,
              preset: QuantizationPreset = QuantizationPreset.PERFORMANCE,
              target_device: TargetDevice = TargetDevice.ANY,
              subset_size: int = 300,
-             fast_error_correction: bool = True,
+             fast_bias_correction: bool = True,
```

Comment:
Was it discussed with @AlexKoff88? As long as I remember we would like to use `fast_error_correction` instead of `fast_bias_correction`.

Reply:
Thanks for the comment! I would suggest to discuss it offline once more and decided naming issue.

Reply:
It was discussed and the idea is to keep the existing name from POT. Frankly, since this API is intended to be used by users who understand DL basics, the word "bias" or "biased model" should be known to them as well. I don't mind changing it to `bias` after all.